test/creating-settings.js

```diff
@@ -110,7 +109,7 @@ describe('Creating throttling settings', () => {
         });
 
         it('should create throttling settings only for the http endpoints with custom throttling configuration', () => {
-          expect(throttlingSettings.endpointSettings).to.have.lengthOf(1);
+          expect(throttlingSettings.endpointSettings).to.have.lengthOf(2);
```

Comment:
The description of this test also needs to be updated. How about: `should create throttling settings only for the http endpoints`.